errata_tool/erratum.py

```diff
@@ -1011,6 +1018,9 @@ def dump(self):
         if self.publish_date_override is not None:
             print('')
             print("Ship Target: {0}".format(self.publish_date_override))
+        elif self.publish_date is not None:
+            print('')
+            print("Ship Target: {0}".format(self.publish_date))
```

Comment:
We're printing two identical "Ship Target" lines here, which seems confusing. How can we distinguish this line from the `print("Ship Target:" ... )` above?

Reply:
From @lhh on IRC, "the `publish_date` is the default one set - perhaps by setting an errata group, or SRT setting an embargo date; the `publish_date_override` is set by the errata creator (devel can edit this one, but not the other)"

Reply:
Ah, I missed this part. I'll update.